sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs

```diff
@@ -441,7 +441,7 @@ private static void CloseLink(RequestResponseAmqpLink link)
                 if (exception is OperationCanceledException &&
                     receiveLink != null && receiveLink.State != AmqpObjectState.Opened)
                 {
-                    // The link state is lost, We need to return a non-retriable error.
+                    // The link state is lost, We need to return a non-retrievable error.
```

Comment:
Why? non-retriable is correct.

Reply:
I literally couldn't parse the word, so I misunderstood what the intent was. My bad. I've never seen it written like that. Seems that it is more popular in our culture now.
https://english.stackexchange.com/a/305433

Reply:
Whooops.   I totally missed that.   Nice catch, Josh.

Reply:
```suggestion
                    // The link state is lost, We need to return a non-retriable error.
```